modules/rosa-sts-aws-requirements-attaching-boundary-policy.adoc

```diff
@@ -0,0 +1,438 @@
+		    "ec2:DescribeVpcEndpointServicePermissions",
+		    "ec2:DescribeVpcEndpointServices",
+		    "ec2:ModifyVpcEndpointServicePermissions"
+            ],
+            "Resource": "*"
+        },
+        {
+            "Effect": "Allow",
+            "Action": [
+                "secretsmanager:GetSecretValue"
+            ],
+            "Resource": "*",
+            "Condition": {
+                "StringEquals": {
+                    "aws:ResourceTag/red-hat-managed": "true"
+                }
+            }
+        }
+    ]
+}
+----
+====
++
+.ROSA installer privatelink permission boundary policy, and policy files
+[cols="1,2",options="header"]
+|===
+|Policy|Description
+
+|`ManagedOpenShift-Privatelink-Permissions-Boundary-Policy`
+|Apply the IAM policy as a permission boundary on a role and it provides the ROSA installer with the permissions required to complete cluster installation tasks.
+
+|===
++
+.`sts_installer_privatelink_permission_boundary_policy.json`
+[%collapsible]
+====
+[source,json]
+----
+{
+"Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Action": [
+        "ec2:ModifyVpcEndpointServiceConfiguration",
+        "route53:ListHostedZonesByVPC",
+        "route53:CreateVPCAssociationAuthorization",
+        "route53:AssociateVPCWithHostedZone",
+        "route53:DeleteVPCAssociationAuthorization",
+        "route53:DisassociateVPCFromHostedZone",
+        "route53:ChangeResourceRecordSets"
+      ],
+      "Resource": "*"
+    }
+  ]
+}
+----
+====
++
+.ROSA installer vpc permission boundary policy, and policy files
+[cols="1,2",options="header"]
+|===
+|Policy|Description
+
+|`ManagedOpenShift-VPC-Permissions-Boundary-Policy``
+|Apply the IAM policy as a permission boundary on a role and it provides the ROSA installer with the permissions required to complete cluster installation tasks.
+
+|===
++
+.`sts_installer_vpc_permission_boundary_policy.json`
+[%collapsible]
+====
+[source,json]
+----
+{
+     "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Action": [
+		    "ec2:AssociateDhcpOptions",
+		    "ec2:AssociateRouteTable",
+		    "ec2:AttachInternetGateway",
+		    "ec2:CreateDhcpOptions",
+		    "ec2:CreateInternetGateway",
+		    "ec2:CreateNatGateway",
+		    "ec2:CreateRouteTable",
+		    "ec2:CreateSubnet",
+		    "ec2:CreateVpc",
+		    "ec2:DeleteDhcpOptions",
+		    "ec2:DeleteInternetGateway",
+		    "ec2:DeleteNatGateway",
+		    "ec2:DeleteRouteTable",
+		    "ec2:DeleteSubnet",
+		    "ec2:DeleteVpc",
+		    "ec2:DetachInternetGateway",
+		    "ec2:DisassociateRouteTable",
+		    "ec2:ModifySubnetAttribute",
+		    "ec2:ModifyVpcAttribute",
+		    "ec2:ReplaceRouteTableAssociation"
+            ],
+            "Resource": "*"
+        }
+    ]
+}
+----
+====
++
+[NOTE]
+====
+Policies pass their json file as a permission boundary in the ROSA (`rosa`) CLI where the installer policy contains a custom boundary.
```

Comment:
If you can, try to avoid anthropomorphism (here, policies are "passing" something). That might involve some passive voice, but possibly something like this:

```suggestion
For each policy, permission boundaries are passed as JSON files in the ROSA (`rosa`) CLI where the installer policy contains a custom boundary.
```